Websites/webkit.org/commit-review.md

```diff
@@ -1,14 +1,14 @@
-The WebKit project has two kinds of special status beyond being a contributor. WebKit Committers have direct read-write access to the Subversion repository, enabling them to commit changes by themselves or reviewed changes by others if authors ask committers to do so. WebKit Reviewers are permitted to review patches and may grant or deny approval for committing. Details of the review and commit process are available on the [contribution page](http://webkit.org/coding/contributing.html).
+The WebKit project has two kinds of special status beyond being a contributor. WebKit Committers have direct read-write access to the repository, enabling them to commit changes by themselves or reviewed changes by others if authors ask committers to do so. WebKit Reviewers are permitted to review patches and may grant or deny approval for committing. Details of the review and commit process are available on the [contribution page](http://webkit.org/coding/contributing.html).
 
 New WebKit Committers and WebKit Reviewers will be selected by the set of existing WebKit Reviewers. We will create a mailing list, &lt;[webkit-reviewers@lists.webkit.org](mailto:webkit-reviewers@lists.webkit.org)&gt;, for this purpose.
 
 An up to date list of current WebKit Committers and WebKit Reviewers will be maintained at webkit.org.
 
 ## Choosing Committers and Reviewers
 
-A candidate for WebKit Committer or WebKit Reviewer should initially be nominated by a reviewer on the reviewers mailing list, in accordance with the criteria below. If the required reviewers (see below) second the nomination, then it carries within 5 business days unless someone objects. If an objection is raised, the reviewers should discuss the matter and try to come to consensus; failing this, the matter will be decided by majority vote of the reviewers.
+A candidate for WebKit Committer or WebKit Reviewer should initially be nominated by a reviewer on the reviewers mailing list, in accordance with the criteria below. If the required reviewers (see below) second the nomination, then it carries in 5 business days unless someone objects. If an objection is raised, the reviewers should discuss the matter and try to come to consensus; failing this, the matter will be decided by majority vote of the reviewers.
 
-Once someone is successfully nominated for WebKit Committer status, Apple will take care of sending the committer agreement and setting up a Subversion account once signed and received.
+Once someone is successfully nominated for WebKit Committer status, Apple will take care of sending the committer agreement and granting read-write access once signed and received.
```

Comment:
Ditto.

Reply:
Currently "read-write access" means that merge-queue will be respected.  That may change in the future (though I doubt anytime soon).  I think this is fine as it is.